queries.sql

```diff
@@ -44,3 +44,22 @@ SELECT neutered, MAX(escape_attempts) AS most_escape FROM animal GROUP BY neuter
 SELECT MIN(weight_kg) AS min_weight, MAX(weight_kg) AS max_weight FROM animal GROUP BY species;
 
 SELECT species, ROUND(AVG(escape_attempts), 2) AS avg_attempt FROM animal WHERE date_of_birth BETWEEN DATE '1990-01-01' AND '2000-12-31' GROUP BY species;
+
+UPDATE animal SET species_id = 2 WHERE name LIKE '%mon';
+
+UPDATE animal SET species_id = 1 WHERE name NOT LIKE '%mon';
```

Comment:
- [ ] Please 🙏🏽 move these lines to the `data.sql` file, it is because everything that populates tables with data should be saved in the data.sql file.

Reply:
@franclobo Will do so